### PR TITLE
add document for aks-acr cross tenant setup

### LINKED ISSUE #427: How to use ACR in AKS from different tenant
Hi,
I have ACR running on tenant A, I would like to attach ACR from  tenant A to Tenant B and Tenant C AKS clusters. I would like to do passwordless image pull on  Tenant B and Tenant C AKS clusters and manage account settings on AAD level.

### PR DESCRIPTION
**Pull request by northtyphoon**
Fix #427

### CODE REVIEW

File: docs/aks-acr-across-tenants.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# AKS and ACR in different tenants
+
```
Comment (dlepow) [inline docs/aks-acr-across-tenants.md:2]: Maybe a title like "Set up AKS to pull from ACR in a different AD tenant"?

File: docs/aks-acr-across-tenants.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+There are several ways to set up the auth credential in Kubernetes to pull image from ACR. For example, you can use [admin user or repository scoped access token](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal) to configure pod [imagePullSecrets](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/). 
+
+While `imagePullSecrets` is commonly used, it brings the challenge and overhead to manage the corresponding secret. On Azure, you can set up [AKS cluster with a service principal credential](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal) which allows you securely pull the image from ACR without additional `imagePullSecrets` setting on each pod.
+
+Sometimes, you may have your AKS and ACR in different Azure Active Directories (Tenants). This document will walk your through the steps to enable cross tenant authentication using service principal credential.
+
+## Instruction
+
+In this example, the AKS cluster is in `Tenant A` and the ACR is in `Tenant B`. 
+
```
Comment (dlepow) [inline docs/aks-acr-across-tenants.md:14]: What permissions does customer need in Tenant A and Tenant B?

File: docs/aks-acr-across-tenants.md (hunk 1 above)
Comment (northtyphoon) [reply docs/aks-acr-across-tenants.md:2]: updated

File: docs/aks-acr-across-tenants.md (hunk 2 above)
Comment (northtyphoon) [reply docs/aks-acr-across-tenants.md:14]: updated

**Review by luisdlp — APPROVED** [review]

**Pull request merged**